suite-native/module-settings/src/components/CryptoUnitsSelector.tsx

```diff
@@ -33,6 +33,7 @@ export const CryptoUnitsSelector = () => {
         >
             <Select<PROTO.AmountUnit>
                 selectValue={bitcoinUnit}
```

Comment:
Since it's related only to Bitcoin, what about renaming the component to `BitcoinUnitsSelector`? 🤔

Reply:
I think why we initially didn't name it Bitcoin name specifically is because it applies to bitcoin-like coins, not only to bitcoin. But I don't have a strong opinion on this and we can rename if you feel like it's better name.